Dshop/apps/users/serializers.py

```diff
@@ -1,11 +1,17 @@
-from django.contrib.auth import authenticate, get_user_model
+from django.contrib.auth import authenticate, get_user_model, password_validation
 from django.utils.translation import gettext_lazy as _
 from rest_framework import serializers
 from rest_framework.serializers import ValidationError
 
+from .models import CustomUser
+
 User = get_user_model()
 
 
+class EmptySerializer(serializers.Serializer):
```

Comment:
Is it used anywhere? If no, please remove.

Reply:
@jacoor 
if i don't use serializer in LogoutView then i receive this warning: 

`D:\Py\dshop\Dshop\Dshop\apps\users\api_views.py: Error [LogoutView]: exception raised while getting serializer. Hint: Is get_serializer_class() returning None or is get_queryset() not working without a request? Ignoring the view for now. (Exception: 'LogoutView' sho
uld either include a `serializer_class` attribute, or override the `get_serializer_class()` method.)
`

I found out on some page that they were using EmptySerializer to handle LogoutView.
What do you think?